go.mod

```diff
@@ -39,4 +39,7 @@ require (
 	sigs.k8s.io/kind v0.11.1
 )
 
-replace github.com/sirupsen/logrus => github.com/projectcalico/logrus v1.0.4-calico
+replace (
+	github.com/projectcalico/api => github.com/song-jiang/api v0.1.2-0.20210913143137-f981580f3483
```

Comment:
do we really want to use the fork here?

Reply:
No. I would update the api pin once my api PR get merged.